website/static/js/share/stats.js

```diff
@@ -121,6 +124,7 @@ Stats.controller = function(vm) {
             url: '/api/v1/share/stats/?' + $.param({q: self.vm.query()}),
             background: true
         }).then(function(data) {
+            data.charts.shareDonutGraph.onclick = function (d, element) { return getSourceName(d, element, self.vm) };
```

Comment:
Can we get access to the vm there? That is originally where we had it, but we weren't sure where the vm was.